client/components/payment-logo/docs/example.jsx

```diff
@@ -17,8 +17,8 @@ const genVendors = flow(
 	filter( type => type !== 'placeholder' ),
 
 	map( type => ( { type, isCompact: false } ) ),
-	concat( [ { type: 'paypal', isCompact: true } ] ),
-	sortBy( [ 'type', 'isCompact' ] )
+	arr => concat( arr, [ { type: 'paypal', isCompact: true } ] ),
+	arr => sortBy( arr, [ 'type', 'isCompact' ] )
```

Comment:
Does this code really work, with the `filter` and `map` calls left as they are? In Lodash, `filter` and `map` always need the collection as the first argument.

This code uses the Ramda/Haskell convention rather than Lodash. There, the `map` function is auto-curried:
```js
const doubler = map( x => x * 2 );
const doubled = doubler( [ 1, 2 ] );
// same as
const doubled = map( x => x * 2, [ 1, 2 ] );
```
For auto-curried functions, there is not difference between `map( fn )( arr )` and `map( fn, arr )`.

Reply:
It doesn't error out anymore, but you're right that it probably doesn't do what's intended. I'll make the fix.

If lodash never did auto-currying, have you got any idea how this ever worked?

Reply:
At least I can see the "compact" version of the PayPal button in the devdocs example:

<img width="367" alt="image" src="https://user-images.githubusercontent.com/87168/59343418-844cd180-8d14-11e9-8047-b3a009577803.png">

While at the site in `/plans` page it looks like this:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/87168/59343365-60898b80-8d14-11e9-9e85-f36d83494c76.png">

Reply:
Applied the same fix to `filter` and `map` and there are multiple logos now, rather than just PayPal. Looks like it was silently producing incorrect behaviour with my initial fix.

Reply:
I had a look at the code and I now see what it was _supposed_ to do 😁 

<img width="489" alt="image" src="https://user-images.githubusercontent.com/87168/59343676-0c32db80-8d15-11e9-91e2-8185fbd42bcc.png">

Reply:
> If lodash never did auto-currying, have you got any idea how this ever worked?

Didn't you convert `lodash/fp/concat` to `lodash` in your `lodash-es` PR? Turns out that the `fp` versions are a bit special :wink: See also https://github.com/Automattic/wp-calypso/pull/24769#discussion_r189493844

Reply:
> Didn't you convert `lodash/fp/concat` to `lodash` in your `lodash-es` PR? Turns out that the `fp` versions are a bit special 😉 See also [#24769 (comment)](https://github.com/Automattic/wp-calypso/pull/24769#discussion_r189493844)

Hah! Mystery solved, good find, @jsnajdr! Of course the `fp` versions are different... sigh. Let's try to avoid relying on internals from now on.